mono/metadata/icall-def.h

```diff
@@ -177,7 +177,9 @@ HANDLES(KPAIR_5, "_ProtectUser", ves_icall_Mono_Security_Cryptography_KeyPairPer
 ICALL_TYPE(APPDOM, "System.AppDomain", APPDOM_23)
 HANDLES(APPDOM_23, "DoUnhandledException", ves_icall_System_AppDomain_DoUnhandledException, void, 2, (MonoAppDomain, MonoException))
 HANDLES(APPDOM_1, "ExecuteAssembly", ves_icall_System_AppDomain_ExecuteAssembly, gint32, 3, (MonoAppDomain, MonoReflectionAssembly, MonoArray))
+#ifndef ENABLE_NETCORE
```

Comment:
Not needed.  This whole file is inside an ifdef.

Reply:
Oh right. Oops.